### PR TITLE
feat: cc-0076 updata migrate script for courtdata

### PR DESCRIPTION
**Pull request by Kornzk**
updata migrate script for six courts

### CODE REVIEW

File: migrations/20220704011115-courtsData.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        width: 15000,
+        centreCircleRadius: 1800,
+        threePointRadius: 6600,
+        threePointLine: 900,
+        lengthOfCorner: 1575,
+        restrictedAreaLength: 5790,
+        restrictedAreaWidth: 4800,
+        sideBorderWidth: 0,
+        lineBorderWidth: 200,
+        description: "This is the default Pro Full Court",
```
Comment (stevenliu986) [inline migrations/20220704011115-courtsData.js:44]: description should be Pro Half Court

File: migrations/20220704011115-courtsData.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        width: 5000,
+        centreCircleRadius: 0,
+        threePointRadius: 6600,
+        threePointLine: 900,
+        lengthOfCorner: 1575,
+        restrictedAreaLength: 5790,
+        restrictedAreaWidth: 4800,
+        sideBorderWidth: 0,
+        lineBorderWidth: 200,
+      },
```
Comment (stevenliu986) [inline migrations/20220704011115-courtsData.js:86]: Plz insert the description

File: migrations/20220704011115-courtsData.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
```
Comment (stevenliu986) [inline migrations/20220704011115-courtsData.js:174]: plz insert description

Comment (Kornzk) [reply migrations/20220704011115-courtsData.js:176]: thanks updated

**Review by stevenliu986 — APPROVED** [review]
LGTM

**Review by EmilyGong111 — APPROVED** [review]
LGTM

**Review by BowenLi61 — APPROVED** [review]

**Pull request merged**